pkg/prometheus/promcfg_test.go

```diff
@@ -473,6 +477,7 @@ func TestProbeIngressSDConfigGeneration(t *testing.T) {
 				Namespace: "default",
 			},
 			Spec: monitoringv1.PrometheusSpec{
+				EnforcedNamespaceLabel: "namespace",
```

Comment:
Could we rather leave this one not set, to make sure its not enforced by default when not set? :) Or do we have another test like this?

Reply:
Yes, let me add another test